Add new document: General Info (#68)
owncloud/docs-ocis: modules/ROOT/pages/deployment/manual/manual-setup.adoc

:toc: right
:toclevels: 2

:downloadpage_ocis_url: https://download.owncloud.com/ocis/ocis/
:systemd-url: https://systemd.io/
:traefik-url: https://doc.traefik.io/traefik/getting-started/install-traefik/

> :description: oCIS can be downloaded from ownCloud or installed via the package manger of your Linux system. Select the way you prefer most. Note that the download version is more often updated than the package manager version.

Reviewer: ```suggestion
:description: oCIS can either be downloaded from ownCloud or installed via the package manger of your Linux system. Use the way you prefer, but keep in mind that the download version is updated more frequently than the package manager version.
```

== Introduction

{description}

> Using the manual method to install oCIS is best when you want to quickly test oCIS without doing additional tasks like docker preperation or kubernetes. Of course you can use it for production in your environment too. See the xref:availability/index.adoc[Availability and Scalability] documentation for impacts. 

Reviewer: ```suggestion
Installing oCIS manually works well if you want to quickly test oCIS without performing additional tasks like Docker preparation or Kubernetes deployment. Of course you can use it for production in your environment, too. See the xref:availability/index.adoc[Availability and Scalability] documentation for impacts. 
```

== Installation

* To get the binary from ownCloud, visit {downloadpage_ocis_url}[download.owncloud.com], select the version and the platform of choice and copy the link of the file. Depending on your system and preferences, you may want to save the binary in `/usr/bin/`.
+
To download use:
+
[source,bash]
----
sudo wget -O /usr/bin/ocis <link>
----
+

> Make the binary executable with

Reviewer: ```suggestion
Make the binary executable with:
```

+
[source,bash]
----
sudo chmod +x /usr/bin/ocis
----
+

> Check the version installed

Reviewer: ```suggestion
Check the version installed:
```

+
[source,bash]
----
ocis --version
----
+

> Which prints an output like

Reviewer: ```suggestion
The output looks like this:
```
